### PR TITLE
Heatmap homepage

### PR DESCRIPTION
**Pull request by edemagbenyo**
Things learnt:
Position with grid

### CODE REVIEW

**Review by Forison — COMMENTED** [review]
# Pull request under review by Addo

**Review by Forison — CHANGES_REQUESTED** [review]
# Good Job Guys on your first attempt, am impressed👌👌
I nearly approved your project😇😇😇
Project well organized and uses semantic HTML, keep it up👌👌👋👋
Here are a few changes I would like us to work on😔
- It contains centered links (content is not important).
> I can't find the centered links in your footer
note > links can be random lorem ipsum text
Screenshot from 2019-09-24 16-11-11


- You should fix the warnings given to you by the[ validator ](https://validator.w3.org)when uploading the index file.🙂
> you can replace the ```article``` with ```div``` or give each ```article``` a heading here you use lorem ipsum for dummy texts. I prefer you to use the second option in this case.

Reminder use more descriptive messages in pr description
## That is it Guys, Do the changes and submit for review.
Happy coding🙂🙂

**Comment by collinsugwu** [comment]
What this PR does
1 a tag added to the links section
2 h tags added to the articles

**Review by stratospherique — CHANGES_REQUESTED** [review]
### Nice job implemented the suggested changes :clap: . However, the fixes that you applied are not exactly what my colleague suggested. check below :arrow_down: 

- You added an `<a>` elements in the footer, however, you need to add some text inside to make them visually apparent (i.e. they are invisible).

- The html validator warnings persists also. :pray: fix them.
  - You can use headings (e.g h2) instead of `<p>` elements inside the article elements.

### That's it :relaxed: . When finished, kindly resubmit another code review request :pray:

File: index.html [hunk 1]
```diff
@@ -1,12 +1,193 @@
...
+        </div>
+        <div class="links">
+            <ul>
+
+                <li><a href="#">One this</a></li>
+                <li><a href="#"> link</a></li>
+                <li><a href="#">to have</a></li>
+                <li><a href="#">access to</a></li>
+                <li><a href="#">the best</a></li>
+                <li><a href="#">posts</a></li>
```
Comment (whiz25) [inline index.html:186]: Screenshot from 2019-10-02 19-52-28


These links still don't appear. Please make them visible and properly centered.

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Good job :+1: 
Please make the suggested change in the comment and re-submit for another PR.

**Review by whiz25 — APPROVED** [review]
## Bravo!
At this point, this project is complete and so I give you a :+1:

**Pull request merged**